nems/util/shr_kind_mod.F90

```diff
@@ -0,0 +1,19 @@
+MODULE shr_kind_mod
```

Comment:
Why is this needed?  Thought you replaced it with med_kinds_mod?

Reply:
see above answer.